mlflow/types/utils.py

```diff
@@ -238,30 +238,6 @@ def _infer_schema(data: Any) -> Schema:
     """
     from scipy.sparse import csc_matrix, csr_matrix
 
-    # List[Dict[str, Union[DataType, List, Dict]]]
```

Comment:
note: Unrelated to this PR, but we found this warning is no-longer required, during the same discussion with PM.